pandas/core/series.py

```diff
@@ -4513,7 +4513,7 @@ def memory_usage(self, index=True, deep=False):
 
     def isin(self, values) -> "Series":
         """
-        Check whether `values` are contained in Series.
+        Whether elements in Series are contained in 'values'.
```

Comment:
Thanks @MarianD , well spotted!

I _think_ it's typical to write variable names with back ticks, so like this:
```suggestion
        Whether elements in Series are contained in `values`.
```